### PR TITLE
Migrate orderbook into coordinator

### PR DESCRIPTION
**Pull request by bonomat**
Follow-up on #155

### CODE REVIEW

File: orderbook/src/tests/sample_test.rs [hunk 1]
```diff
@@ -1,115 +0,0 @@
-use crate::models::Order;
```
Comment (bonomat) [inline orderbook/src/tests/sample_test.rs:1]: will be migrated separately in the next commit.

Comment (1010Tom) [inline orderbook/src/tests/sample_test.rs:1]: Will be migrated in the next commit.

File: .gitignore [hunk 2]
```diff
@@ -28,3 +28,6 @@ mobile/ios/Runner/libnative.a
 # ignore data runtime data
 data/**
 !data/coordinator/regtest/seed
+
+# Postgres db data
+pgdata
```
Comment (Restioson) [inline .gitignore:33]: nit: no newline at end of file

**Review by Restioson — APPROVED** [review]
LGTM I think!

**Review by holzeis — COMMENTED** [review]
It's hard for me to properly review that PR as I haven't reviewed the initial orderbook PR. Also I can't compile the coordinator due to this error.

```
note: ld: library not found for -lpq
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by holzeis** [comment]
I found that, but it also did not work for me. https://github.com/diesel-rs/diesel/blob/master/guide_drafts/backend_installation.md

**Review by holzeis — APPROVED** [review]
Ok, besides installing postgres and libpq I had to also run `cargo clean`. Now it's working for me. The coordinator was able to start! LGTM!

**Pull request merged**